wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

```diff
@@ -50,7 +50,8 @@ All game modes use a grading system similar to the one in osu!mania on stable (c
 | A | At least 90% accuracy |
 | B | At least 80% accuracy |
 | C | At least 70% accuracy |
-| D | Anything else |
+| D | Anything else with passed score |
+| F | Failed score |
```

Comment:
this mirrors https://osu.ppy.sh/wiki/en/Gameplay/Grade, which doesn't display F grades. they don't exist anywhere except in lazer when you save a failed score

Reply:
than should we keep this or remove?
We may need update grade to match this?

F scores are also exist on the server side, but they cannot be obtained by methods other than api.

Reply:
i'll revert for now, but this can be considered in another pr (mentioning that it's only used in the api, etc.)